client/my-sites/exporter/exporter.jsx

```diff
@@ -52,8 +53,16 @@ export default React.createClass( {
 		} = this.props;
 		const siteId = this.props.site.ID;
 
-		const exportAll = () => startExport( siteId );
-		const exportSelectedItems = () => startExport( siteId, { exportAll: false } );
+		const exportAll = () => {
+			analytics.tracks.recordEvent( 'calypso_export_start_click', { scope: 'all' } );
```

Comment:
Haha, agreed `calypso_export_start_button_click` makes more sense.

> I also wondered if we shouldn't move the call to `recordEvent()` into `startExport()`

I thought about this, but the action creator is global and could be used in other places. I think if we were to do that, the tracks name should be just `calypso_export_start`, since it could be started from somewhere else (on the delete site page for example).